app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -246,7 +246,7 @@
 }
 // modal overflow scrolling
 .onboarding-modal-scroll-container {
-  height: calc(100vh - 289px);
+  height: calc(100vh - 332px);
```

Comment:
Since it was causing two scrollbars I changed it to so that it has only one scrollbar